packages/wasmer/wasmer.1.2.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+
+authors: ["rajdakin"]
+maintainer: ["engineering@wasmer.io"]
+license: "MIT"
+homepage: "https://github.com/wasmerio/wasmer-ocaml"
+bug-reports: "https://github.com/wasmerio/wasmer-ocaml/issues"
+dev-repo: "git+https://github.com/wasmerio/wasmer-ocaml.git"
+tags: ["org:wasmer"]
+
+url {
+	src: "https://github.com/wasmerio/wasmer-ocaml/archive/refs/tags/v1.2.1+dunefix.tar.gz"
+	checksum: [
+		"md5=be37208390097ed6b7c7d9ede3c1a2a7"
+		"sha512=2422c3b32ae053a9f39336b1c9829ec9a508dd7d303e9a68853287ab7dea90b7cbc57f54a04b20a9fb6217527e1c89e8f806349407db5f8679d90fdf7dd3038e"
+	]
+}
+
+synopsis: "OCaml bindings for Wasmer"
+description: "The official Wasmer bindings for OCaml"
+
+depends: [
+  "ocaml" {>= "4.14.1"}
```

Comment:
OCaml 4.14.1 hasn't been released yet
```suggestion
  "ocaml" {>= "4.14.0"}
```

Reply:
Yes, but OCaml 4.14.0 has a bug that makes the program crash when exiting (see ocaml/ocaml#11489). It has been fixed only for OCaml 5.0 and 4.14.1, but 4.14.0 still has this bug.

Reply:
ah my bad. Good to go then.